apps/testrunner/test_suites/simple.py

```diff
@@ -25,13 +26,26 @@ def __init__(self, *args, **kwargs):
         self.TARGET_ENV = test_runner_config['target_hosts'][0]['hostname']
 
     def run(self, **kwargs):
-        retries = kwargs.pop('retries')
```

Comment:
WE actually could leave it like that:
```python 
retries = kwargs.pop('retries', self.DEFAULT_RETRIES_COUNT)
```
now we can write
```python
retries = kwargs.get('retries', self.DEFAULT_RETRIES_COUNT)
```
Purpose of this change is to get retries parameter in logged output?